keps/sig-release/20190316-rebase-images-to-distroless.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Rebase Kubernetes Main Master and Node Images to Distroless
+authors:
+  - "@yuwenma"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
```

Comment:
Add sig-cloud-provider here. I think you can remove sig-auth too.

Reply:
Weird. Somehow this is the old commit. Updated.